src/components/autosuggest/example-autosuggest-country.njk

```diff
@@ -5,6 +5,7 @@
 
         {{ onsAutosuggest({
             "id": "country-of-birth",
+            "width": 10,
```

Comment:
I think this looks a bit too narrow in the example. To demonstrate this I would set it to 30 or something